perses/app/relative_setup.py

```diff
@@ -122,27 +122,21 @@ def __init__(self,
             box dimensions for the complex phase
         solvent_box_dimensions: Vec(3), optional, default=None
             box dimensions for the solvent phase
-        map_strategy : 'str' default='geometry'
-            determines which map is considered the best and returned
-            options are 'geometry', 'random', 'weighted-random', 'return-all'
-            can be one of ['geometry', 'matching_criterion', 'random', 'weighted-random', 'return-all']
-            - core will return the map with the largest number of atoms in the core. If there are multiple maps with the same highest score, then `matching_criterion` is used to tie break
-            - geometry uses the coordinates of the molB oemol to calculate the heavy atom distance between the proposed map and the actual geometry
-            this can be vital for getting the orientation of ortho- and meta- substituents correct in constrained (i.e. protein-like) environments.
-            this is ONLY useful if the positions of ligand B are known and/or correctly aligned.
-            - matching_criterion uses the `matching_criterion` flag to pick which of the maps best satisfies a 2D requirement.
-            - random will use a random map of those that are possible
-            - weighted-random uses a map chosen at random, proportional to how close it is in geometry to ligand B. The same as for 'geometry', this requires the coordinates of ligand B to be meaninful
-            - return-all BREAKS THE API as it returns a list of dicts, rather than list. This is intended for development code, not main pipeline.
```

Comment:
Wouldn't removing these would affect the expected behavior of some current/older code or uses?

Reply:
I changed all uses of `map_strategy` within `perses`, but if there are external users (or unmerged branches), these would need to update to the new API. But this can't be helped, and we're still pre-1.0 without a stable API.